docs/content/docs/install/manual_demo/_index.md

```diff
@@ -20,9 +20,29 @@ The OSM Manual Install Demo Guide is a step by step set of instructions to quick
   - `git clone https://github.com/openservicemesh/osm.git`
   - `cd osm`
 
-## Build or Download the OSM CLI
 
-Use the [installation guide](../../install) to install the `osm` cli.
+## Download and install the OSM command-line tool
+
+The `osm` command-line tool contains everything needed to install and configure Open Service Mesh.
+The binary is available on the [OSM GitHub releases page](https://github.com/openservicemesh/osm/releases/).
+
+Download the 64-bit GNU/Linux or macOS binary of OSM v0.8.2:
```

Comment:
are we going to bump this version/doc with each release?

Reply:
That would be the goal.
This is a bit of pain when we switch to versioned docs, but that pain on our side will be a seamless experience for the folks reading these docs and using OSM.
These should also be reflected here:
![image](https://user-images.githubusercontent.com/49918230/112561283-27f45e00-8d92-11eb-8684-1cc3b3aa448d.png)


@SanyaKochhar how do you think about it?
We could perhaps not include patch and keep these at `v0.8`?